src/js/Models/index.js

```diff
@@ -0,0 +1,7 @@
+import { dispatch as dispatchGlobal } from './global/global.js';
+import { dispatch as dispatchRace } from './race/race.js';
+
+export function resetModels() {
+  const models = [dispatchGlobal, dispatchRace];
+  models.forEach((dispatch) => dispatch());
```

Comment:
```suggestion
  dispatchGlobal();
  dispatchRace();
```
이렇게 더 간단하게 표현할 수 있지 않을까요!?

Reply:
```js
dispatchGlobal('reset');
dispatchRace('reset');
```

```js
dispatchGlobal({ type: 'reset' });
dispatchRace({ type: 'reset' });
```
그리고 이렇게 명시적으로 reset이라는 행위를 정의해놓고 사용하면 어떨까요?

Reply:
훨씬 명시적이고 좋은 것 같네요! 감사합니다!